src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -1,155 +1,156 @@
-<div class="center">
```

Comment:
Please update your version first before pulling it to the Repository. 
This messed up the view format.